omod-2.0/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/wrapper/AdministrationSectionLinks2_0.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.wrapper;
+
+import org.openmrs.module.web.extension.AdministrationSectionExt;
+
+import java.util.Map;
+
+/**
+ * REST wrapper for {@link AdministrationSectionExt}
+ */
+public class AdministrationSectionLinks2_0 {
```

Comment:
Why the name `AdministrationSectionLinks2_0`?

Reply:
I've seen it's a convention of appending resource version to wrapper class names. 
For example: `CohortMember1_8, IncomingHl7Message1_8, UserAndPassword1_8`

Reply:
For resources that work only in specific openmrs platform versions. But for one like this which can work in all the supported ones, it is not necessary.

Reply:
Resolved here: https://github.com/openmrs/openmrs-module-webservices.rest/pull/484/commits/d5cba97f125983e66074d932555ab3dce13ddde3